jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/PdfModule.java

```diff
@@ -109,21 +110,274 @@
+	private static final String PDF_SIG_HEADER = "%" + PDF_VER1_HEADER_PREFIX;
+	private static final String POSTSCRIPT_HEADER_PREFIX = "!PS-Adobe-";
+	private static final String ENCODING_PREFIX = "ENC=";
+    private static final String NO_HEADER = "No PDF header";
+
+	private static final String DEFAULT_PAGE_LAYOUT = "SinglePage";
+	private static final String DEFAULT_MODE = "UseNone";
+
+	private static final String FILTER_NAME_CRYPT = "Crypt";
+	private static final String FILTER_VAL_STANDARD = "Standard";
+
+	private static final String RESOURCE_NAME_XOBJECT = "XObject";
+
+	private static final String FONT_TYPE0 = "Type0";
+	private static final String FONT_TYPE1 = "Type1";
+	private static final String FONT_TYPE3 = "Type3";
+	private static final String FONT_MM_TYPE1 = "MMType1";
+	private static final String FONT_TRUE_TYPE = "TrueType";
+	private static final String FONT_CID_TYPE0 = "CIDFontType0";
+	private static final String FONT_CID_TYPE2 = "CIDFontType2";
+
+	private static final String ACTION_VAL_GOTO = "GoTo";
+
+	private static final String DICT_KEY_DIRECTION = "Direction";
+
+	private static final String DICT_KEY_CENTER_WINDOW = "CenterWindow";
+	private static final String DICT_KEY_DISP_DOC_TITLE = "DisplayDocTitle";
+	private static final String DICT_KEY_FIT_WINDOW = "FitWindow";
+	private static final String DICT_KEY_HIDE_MENUBAR = "HideMenubar";
+	private static final String DICT_KEY_HIDE_TOOLBAR = "HideToolbar";
+	private static final String DICT_KEY_HIDE_WINDOW_UI = "HideWindowUI";
+	private static final String DICT_KEY_NO_FULL_PAGE = "NonFullScreenPageMode";
+	private static final String DICT_KEY_PAGE_CLIP = "PageClip";
+	private static final String DICT_KEY_PRINT_AREA = "PrintArea";
+	private static final String DICT_KEY_VIEW_AREA = "ViewArea";
+	private static final String DICT_KEY_VIEW_CLIP = "ViewClip";
+
+	private static final String PROP_NAME_CENTER_WINDOW = DICT_KEY_CENTER_WINDOW;
+	private static final String PROP_NAME_DISP_DOC_TITLE = DICT_KEY_DISP_DOC_TITLE;
+	private static final String PROP_NAME_FIT_WINDOW = DICT_KEY_FIT_WINDOW;
+	private static final String PROP_NAME_HIDE_MENUBAR = DICT_KEY_HIDE_MENUBAR;
+	private static final String PROP_NAME_HIDE_TOOLBAR = DICT_KEY_HIDE_TOOLBAR;
+	private static final String PROP_NAME_HIDE_WINDOW_UI = DICT_KEY_HIDE_WINDOW_UI;
+	private static final String PROP_NAME_NO_FULL_PAGE = DICT_KEY_NO_FULL_PAGE;
+	private static final String PROP_NAME_PAGE_CLIP = DICT_KEY_PAGE_CLIP;
+	private static final String PROP_NAME_PRINT_AREA = DICT_KEY_PRINT_AREA;
+	private static final String PROP_NAME_VIEW_AREA = DICT_KEY_VIEW_AREA;
+	private static final String PROP_NAME_VIEW_CLIP = DICT_KEY_VIEW_CLIP;
+	private static final String PROP_NAME_DIRECTION = DICT_KEY_DIRECTION;
+
+	private static final String DICT_KEY_FONT_DESCRIPTOR = "FontDescriptor";
+	private static final String DICT_KEY_STARTXREF = "startxref";
+	private static final String DICT_KEY_BASE_FONT = base + font;
```

Comment:
This code would be easier to read as a literal "BaseFont", cutting out the time it takes to confirm the contents of the variables, and allowing the key to be searched. They're also variables that are unlikely to need changing in the future, so I see little benefit in having them as variables. I think the same can be said for most of the variables declared on lines 116 to 121, with the exception of `crop_box`.